packages/mobile/src/screens/profile-screen/BuyArtistCoinButton.tsx

```diff
@@ -0,0 +1,54 @@
+import { useCallback } from 'react'
+
+import { useArtistOwnedCoin, useProfileUser } from '@audius/common/api'
+import { useFeatureFlag } from '@audius/common/hooks'
+import { FeatureFlags } from '@audius/common/services'
+
+import { Button, useTheme } from '@audius/harmony-native'
+import { useNavigation } from 'app/hooks/useNavigation'
+
+const messages = {
+  title: 'Buy Artist Coin'
+}
+
+export const BuyArtistCoinButton = () => {
+  const { color } = useTheme()
+  const navigation = useNavigation()
+  const { isEnabled: isArtistCoinsEnabled } = useFeatureFlag(
+    FeatureFlags.ARTIST_COINS
+  )
+
+  const { user_id } =
+    useProfileUser({
+      select: (user) => ({
+        user_id: user.user_id
+      })
+    }).user ?? {}
```

Comment:
wat... it's selecting user_id but then doing .user on user_id??

Reply:
yah wth